docs/user-manual/consume.md

```diff
@@ -5,25 +5,73 @@ slug: /consume
+has a [reference input][reference-1] encoded as an [inline datum][datum-1].
+
+Reference inputs are published using a COOP authentication token with a COOP
+minting policy identifier, currently:
+
+* preprod: `5ec8416ecd8af5fe338068b2aee00a028dc1f4c0cd5978fb86d7c038?`
+* mainnet: `84da51fd8041b6a1073f1154ddde0c846d71c4081fc1f7ba66e0e6b5?`
+
+This is important as fact statements should be verified as coming from a known
+source and this also helps to filter all of the UTxO at a given smart contract
+address where anyone has the ability to send other UTxO.
+
+Users will therefore need to use the following information to identify the
+latest facts.
+
+* Minting policy ID.
+* Datum format, including:
+  * Most recent (`ValueReference -> PropertyValue[1] -> value)`, i.e. the
+  largest POSIX timestamp compared to other fact statement datum,
+  * Feed name, e.g. "ADA-USD".
+
+> Note: Users on preprod will, for example, find a volume of test data that
+> hasn't been retired that needs to be ignored using the latest preprod policy
+> ID and fact schema.
+
+Unconsumed reference inputs are those still available for smart contracts to use
+on-chain. Spending a UTxO with a reference input renders it unusable in a smart
+contract (though the data will be visible in the historical transaction).
+Orcfax's mainnet V1 will operate a policy of two unspent fact UTxO remaining
+on-chain at all times. Consuming reference inputs will sometimes be referred to
+as garbage collection or *retiring* of fact statements.
+
+For more information about the use of reference input and inline datum in COOP
+see COOP's [design document][coop-design-1].
+
+[coop-design-1]: https://github.com/mlabs-haskell/cardano-open-oracle-protocol/blob/9e9c9aedba84d32e424b1dd116b4734e1a42f3bc/coop-docs/00-design.md#cardano-features-enabling-oracles
+
+[utxo-1]: https://docs.cardano.org/learn/eutxo-explainer/
+[reference-1]: https://docs.cardano.org/cardano-testnet/about/feature-overview/#referenceinputs(cip-31)
+[datum-1]: https://docs.cardano.org/cardano-testnet/about/feature-overview/#inlinedatums(cip-32)
+
+## Datum structure
+
+All fact statements are structured as [JSON-LD](https://json-ld.org/) objects
 before they are serialized into the Concise Binary Object Representation
 ([CBOR](https://cbor.io/)) that is used in Cardano transactions.
 
 ![Fact Statement](/img/2023-09-30--Orcfax--fact-statement.jpg)
 
-
 ## Read CBOR datum on-chain
+
 Orcfax fact statement datum are readable on-chain as Cardano
 [Reference Inputs](https://github.com/perturbing/vasil-tests/blob/main/reference-inputs-cip-31.md).
 
 To read our `ADA-USD|USD-ADA` Mainnet feed, check the latest transaction
 issued by the [feed smart contract](https://cexplorer.io/address/addr1w8tcecfy7np3sduzn99ffuv8qx2sa8v977l0xql8ca7lgkgq7lqh2/tx#data).
 
 ## Read CBOR datum off-chain
+
 The [PyCardano](https://pycardano.readthedocs.io/) SDK provides convenient
 helper functions for converting on-chain CBOR.
 
```

Comment:
Change, "It is made freely available under an open-source Apache v2.0 license..." to
This resource has been made freely available under..."